### PR TITLE
refactor(dialog): improve dialog

### PR DESCRIPTION
**Pull request by matmkian**
# General summary

See changelog

# Screenshots

# Check list

-   [x] (if has code) Add `need: review-frontend` label
-   [x] (if has react) Check through the [react dev check list]
-   [x] (if fix or feature) Add `need: test` label

Check out the [contribution guide] for general guidelines for submissions to the design system.

[contribution guide]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md
[react dev check list]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md#react-developpment-check-list

### CODE REVIEW

**Comment by matmkian** [comment]
Your turn @gcornut :-)

File: packages/lumx-react/src/components/dialog/Dialog.test.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                continue;
+            }
+
+            it(`should render story ${storyName}`, () => {
+                // @ts-ignore
+                const wrapper = shallow(<Story />);
+                expect(wrapper).toMatchSnapshot();
+            });
+        }
+    });
```
Comment (gcornut) [inline packages/lumx-react/src/components/dialog/Dialog.test.tsx]: @juanigalan91 @pidupuis One way to do snapshot testing on stories without any storybook or Jest addons

Comment (juanigalan91) [reply packages/lumx-react/src/components/dialog/Dialog.test.tsx]: ah nice :) definitely a great way to avoid repeating props on both stories and tests !

**Comment by matmkian** [comment]
Test KO -> dropdown should disappear with a fadeout animation

**Pull request merged**